composer/translate-composer.js

```diff
@@ -1371,6 +1374,7 @@ var TranslateComposer = exports.TranslateComposer = Composer.specialize(/** @len
             translateEvent.initCustomEvent("translate", true, true, null);
             translateEvent.translateX = this._translateX;
             translateEvent.translateY = this._translateY;
+            translateEvent.targetElement = this.element;
```

Comment:
Is this to workaround the fact that today the target is the translate composer itself?

Reply:
yes